_posts/2019-05-03-tadpoles-on-tensorflow/tadpoles-on-tensorflow.Rmd

```diff
@@ -0,0 +1,681 @@
+---
+title: "Tadpoles on TensorFlow: Hierarchical partial pooling with tfprobability"
+description: > 
+ This post is a first introduction to MCMC modeling with tfprobability, the R interface to TensorFlow Probability (TFP). Our example is a multi-level model describing tadpole mortality, which may be known to the reader from Richard McElreath's wonderful "Statistical Rethinking".  
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydana2019tadpoles
+date: 05-03-2019
+categories:
+  - Probability and statistics 
+  - Introductions
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/thumb.png
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+So before we jump into the technicalities, -- yes, that's what this post is too: _Un hommage à_ McElreath who wrote one of most intriguing books on Bayesian (or should we just say - scientific?) modeling we're aware of. If you haven't read _Statistical Rethinking_, and are interested in modeling, you might definitely want to check it out. In this post, we're not going to try to re-tell the story: Our clear focus will instead be a demonstration of how to do MCMC with [tfprobability](https://rstudio.github.io/tfprobability/).
+
+Concretely, this post has two parts. The first is a quick overview of how to use `tfd_joint_sequential_distribution` to construct a model, and then sample from it using Hamiltonian Monte Carlo. This part could be consulted for quick code look-up, or as a frugal template of the whole process.
```

Comment:
Link to `tfd_join_sequential_distribution` function reference, if you have it?

Reply:
"This part could be…" I think you can use _can_ here (can be consulted…)

Reply:
right & right!